### PR TITLE
HDDS-4460. Add metrics for closed container replication

### PR DESCRIPTION
**Pull request by elek**
## What changes were proposed in this pull request?

Today it's hard to understand what's going on and what is the performance of the replication.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4460

## How was this patch tested?

 1. Started docker compose cluster
 2. Generated keys
 3. Removed one datanode
 4. Restored one datanode
 5. Waited until data is replicated
 6. Checked the metrics

### CODE REVIEW

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/MeasuredReplicator.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    DefaultMetricsSystem.instance()
+        .register(NAME, "Closed container replication", this);
+  }
+
+  @Override
+  public void replicate(ReplicationTask task) {
+    long start = System.currentTimeMillis();
+
+    long msInQueue =
+        Instant.now().getNano() - task.getQueued().getNano() / 1_000_000;
```
Comment (jojochuang) [inline hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/MeasuredReplicator.java:67]: did you forget a pair of brackets here? Like 
```
(Instant.now().getNano() - task.getQueued().getNano()) / 1_000_000;
```

**Review by jojochuang — DISMISSED** [review]
the rest looks good

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/MeasuredReplicator.java (hunk 1 above)
Comment (elek) [reply hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/MeasuredReplicator.java:67]: OMG.

Fixed, thanks to save me ;-)

**Review by adoroszlai — APPROVED** [review]
Some ideas for future improvement:

* bytes transferred for failed tasks
* time spent on failed attempts
* failure reason (download or import - unless these steps are merged)

**Comment by adoroszlai** [comment]
Thanks @elek for the patch and @jojochuang for the review.

**Pull request merged**